apps/nuxt3-ssr/components/SearchResultsViewTabsMobile.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
```

Comment:
I have seen many mobile now. Was that what we agreed to do? 

More importantly, do we now need implement all views multiple times?

Reply:
only those that need more then flex allows for , depends on the quality needed for the 'mobile' view.